### ISSUE TITLE
[Feature Request, Meta]: Enable 3rd party OAuth app access

### ISSUE THREAD

**Issue by MasterOfBob777** [FTBTeam/FTB-Mods-Issues#135]
### Mod

Meta (the FTBTeam github organization)

### Description

Enable 3rd party OAuth app access to repositories. Specfically [Codestream](https://marketplace.visualstudio.com/items?itemName=CodeStream.codestream), this would increase (my, and others) productivity when making a pull request or commit to a FTB repository. For me, I do all of my github access in vscode so restricting this hinders me workflow.  

### Describe the solution you'd like

[Enabling OAuth app access](https://docs.github.com/en/organizations/restricting-access-to-your-organizations-data/enabling-oauth-app-access-restrictions-for-your-organization).

### Describe alternatives you've considered

I understand this is most likely because of security concerns, but this problem should be solving in a better way. Completely disabling 3rd party apps just discourages outside contributions to your repositories.
So in other words, not accepting the feature, in my eyes  it's mostly black and white, this only restricts a few people among the hundreds that would likely want to commit to FTB repositories, while only minorly inconveniencing them.

**Comment by Jake-E** [comment]
I have allowed CodeStream as a third-party tool, however, we will not be completely opening up all third-party apps.
